Scripts/NuGet/EdFi.Installer.WebApi/test.ps1

```diff
@@ -44,6 +44,21 @@ function Invoke-NonDefaultApplication {
     Install-EdFiOdsWebApi @p
 }
 
+function Invoke-DifferentPackageSource {
+    $p = @{
+        ToolsPath = "../../../tools"
+        InstallType = "SharedInstance"
+        DbConnectionInfo = @{
+            Engine="SqlServer"
+            Server="localhost"
+            UseIntegratedSecurity=$true
+        }
+        PackageName = "EdFi.Suite3.Ods.WebApi"
+        PackageVersion = "5.2.14321"
```

Comment:
As long as your new package is the most recent we shouldn't need the version here. Since the package name defaults to the value you've set here I don't see a reason to specify it explicitly.

Reply:
I updated it  .Thanks